### PR TITLE
ADAPT-1073: Unstructured WYSIWYG content styles as TW component

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Provide some basic vertical rhythm for unstructured WYSIWYG content with the .su-wysiwyg class
- I mostly moved over what's been tested on the Giving site.

# Needed By (Date)
- When does this need to be merged by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Look at preview page, click on "WYSIWYG Text" pill. Look at WYSIWYG content section
2. Look at code

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: src/plugins/components/typography/wysiwyg.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          'a': {
+            textDecoration: 'none',
+          },
+          'a:hover, a:focus': {
+            textDecoration: 'underline',
+          }
+        },
+        'h2 + h3, h3 + h4, h4 + h5, h5 + h6': {
+          '&:not(:first-child)': {
+            marginTop: '1em',
```
Comment (sherakama) [inline src/plugins/components/typography/wysiwyg.js:38]: Margin tops? Porque?

Comment (yvonnetangsu) [reply src/plugins/components/typography/wysiwyg.js:38]: That's for when headings stacked inside WYSIWYG 😃

File: src/plugins/components/typography/wysiwyg.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          marginBottom: '1.6em',
+        },
+        'li p': {
+          marginBottom: '0.4em',
+        },
+        'h2, h3, h4, h5, h6': {
+          marginBottom: '0.7em',
+          clear: 'both',
+          '&:not(:first-child)': {
+            marginTop: '1.6em',
```
Comment (sherakama) [inline src/plugins/components/typography/wysiwyg.js:27]: Perhaps I should have called out this one. Just want to call it out to make sure that having margin tops in wysiwyg is expected.

Comment (yvonnetangsu) [reply src/plugins/components/typography/wysiwyg.js:27]: Yup this has been tested. When there's a heading, I want to give it more space above so there's a larger separation between the heading and its previous element, and this looks better if it's proportional to the heading size (thus using em also). This is also due to the limitation of CSS - there's no way to select a previous element to give it a margin bottom (plus then it won't be proportional to the heading size).

**Review by sherakama — APPROVED** [review]
GTG

**Pull request merged**